.devcontainer/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.177.0/containers/javascript-node/.devcontainer/base.Dockerfile
+
+# [Choice] Node.js version: 16, 14, 12
+ARG VARIANT="16-buster"
+FROM mcr.microsoft.com/vscode/devcontainers/javascript-node:0-${VARIANT}
+
+# [Optional] Uncomment this section to install additional OS packages.
+# RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
```

Comment:
can we install DVC here? 

and the VS Code extension (probably we would need some cli command to install from VSIX)

Reply:
> can we install DVC here?

Is it needed for the VCS extension? Otherwise maybe contributors should have `dvc` locally.

> and the VS Code 

But there doesn't seem to be a defined place to get the VSIX file programmatically? You have to go to https://github.com/iterative/vscode-dvc/actions/, choose a passing build (unclear which one will be more stable), and download the artifact (and unzip).

Reply:
We'll probably need to do a real publish before we install the VSCode extension in this thing. Maybe we can get away with it if we mark the Extension "pre-alpha"? Installing it in codespaces would effectively be making it public anyway.

Reply:
I guess I'll try this separately in an actual DVC repo like https://github.com/iterative/example-get-started. It may make more sense there.